docs/source/receptorctl/receptorctl_work_submit.rst

```diff
@@ -33,13 +33,29 @@ Work submit options
 
 You can use the following options with the ``work submit`` command:
 
-``-a``, ``--param <<KEY>>=<<VALUE>>`` adds a Receptor parameter in key=value format.
-``-f``, ``--follow`` keeps Receptorctl to remain attached to the job and displays the job results.
-``-l``, ``--payload-literal <<TEXT>>`` uses the value of ``<<TEXT>>`` as the literal unit of work data.
-``-n``, ``--no-payload`` sends an empty payload.
-``--node <<Node ID>>`` is the Receptor node on which the work runs. The default is the local node.
-``-p``, ``--payload <<TEXT>>`` specifies the file that contains data for the unit of work. Specify ``-`` for standard input (stdin).
-``--rm`` releases the work unit after completion.
-``--signwork`` digitally signs remote work submissions to standard output (stdout).
-``--tls-client <<TEXT>>`` specifies the TLS client that submits work to a remote node.
-``--ttl <<TEXT>>`` specifies the time to live (TTL) for remote work requests in ``##h##m##s`` format; for example ``1h20m30s`` or ``30m10s``. Use the ``receptorctl work list`` command to display units of work on Receptor nodes and determine appropriate TTL values.
+.. list-table:: Options
```

Comment:
```suggestion
.. list-table::
```

Reply:
It seems redundant to have a title for the table here, especially so close to the actual previous heading. I'd either remove this or expand it to something like "Options for the work submit command."

I think the whole point of the table title is for accessibility purposes so a longer, more descriptive title is better. However it's [not strictly necessary](https://webaim.org/techniques/tables/data) to have a title / caption. You've got the "Option | Description" headers, which is the main thing. I think the surrounding text also provides adequate context.